app/_components/Home/Gallery/index.jsx

```diff
@@ -1,9 +1,68 @@
+import Marquee from 'react-fast-marquee';
+import galleryJson from '@/app/data/home/gallery.json';
+
 // TODO: Create a Gallery component.
 export default function Gallery() {
-    return (
-      <main>
-        <h1 id="gallery" className="text-6xl font-bold">Gallery</h1>
-      </main>
-    );
-  }
-  
\ No newline at end of file
+  return (
+    <main>
+      <h1 id="gallery" className="text-6xl font-bold text-center">
+        Gallery
+      </h1>
+
+      <div className="gallery space-y-4 py-16 container block mx-auto overflow-x-hidden">
+        <div className="flex overflow-x-hidden">
+          <Marquee
+            gradient={false}
+            speed={40}
+            direction="left"
+            className="flex"
```

Comment:
`flex` within an element that already has `flex` is redundant.

Reply:
Resolved